edit-post/components/header/plugins/style.scss

```diff
@@ -0,0 +1,42 @@
+.components-choice-menu-plugins {
+	width: 100%;
+	padding: 10px;
+}
+
+.components-choice-menu-plugins__label {
+	margin-bottom: 10px;
+	color: $dark-gray-300;
+}
+
+.components-menu-item-plugins__button,
```

Comment:
This is almost the same as `.components-menu-items__button`, but it adds some modifications. Question is if we can reuse the existing class or even better an existing component?